server/src/db.ts

```diff
@@ -21,18 +21,36 @@ import { existsSync, mkdirSync } from 'fs';
 import { join, dirname } from 'path';
 import { getEnvironmentVariable } from './util';
 import { init as initDB, dbRun, dbAll } from './sqlite';
-import { Zone, CreateZoneRequest } from './common/types';
+import {
+  Zone,
+  CreateZoneRequest,
+  LightType,
+  Light,
+  CreateLightRequest,
+  RVLLight,
+  CreateRVLLightRequest,
+  HueLight,
+  CreateHueLightRequest
+} from './common/types';
 
 const DB_FILE = join(
   getEnvironmentVariable('HOME'),
   '.homelights',
   'db.sqlite3'
 );
 
-const ROOM_SCHEMA = `
+const ZONE_SCHEMA = `
 CREATE TABLE "zones" (
   id INTEGER PRIMARY KEY AUTOINCREMENT,
-  name text NOT NULL UNIQUE
+  name TEXT NOT NULL UNIQUE
+)`;
+
+const LIGHT_SCHEMA = `
+CREATE TABLE "lights" (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  name TEXT NOT NULL UNIQUE,
+  type TEXT NOT NULL,
```

Comment:
I'd love to insert an enum table for this

Reply:
My first though too, but SQLite doesn't support enums. I'm not worried about it though, since it's backed by an enum in TypeScript, and so will have the same effect.